main/solution/backend/config/settings/.defaults.yml

```diff
@@ -289,3 +289,12 @@ egressStoreKmsPolicyWorkspaceSid: Allow workspace environments to access egress
 # The name of the IAM role created for the Lambda egressStoreObjectsHandler
 egressStoreObjectHandlerRoleName: ${self:custom.settings.namespace}-EgressStoreObject
 # =========================================================================================================
+
+# ================================ DevOps Role details ===========================================
+# Stack name of the prep-devops component
+# Expanding global namespace since the settings expander only supports upto two levels currently
+prepDevopsStackName: prep-devops-account-${self:custom.settings.awsAccountInfo.awsAccountId}-${self:custom.settings.namespace}
+
+# ExternalID to assume the above role
+devopsRoleExternalId: ${self:custom.settings.awsAccountInfo.awsAccountId}
```

Comment:
External ID is usually an optional string value associated to the account. How are we passing the account ID here?

Reply:
External ID is used as an additional identifier when another AWS account is trying to assume an IAM role. We are using the base account's account ID as the identifier. But that can be changed to any string if needed.